holmes/plugins/toolsets/argocd.yaml

```diff
@@ -0,0 +1,37 @@
+toolsets:
+  - name: "argocd/core"
+    prerequisites:
+      - command: "argocd version"
+
+    tools:
+      - name: "argocd_app_get"
+        description: "Retrieve information about an existing application, such as its status and configuration"
+        command: "argocd app get {{ app_name }} --show-operation -o yaml --auth-token {{ auth_token }}"
```

Comment:
I think we don't want to auth token to be sent to the llm
Can we somehow pass the token via env var?
So, for example, in order to use the tool, the user need to inject the env var: ARGO_AUTH_TOKEN,
and the tool command will be:
```
command: "argocd app get {{ app_name }} --show-operation -o yaml --auth-token $ARGO_AUTH_TOKEN"
```
Can something like this work?

Reply:
Turns out we don't event need to use `--auth-token` as argocd will read the env var `ARGO_AUTH_TOKEN` by itself. Reflected in the documentation.